webapp/src/constants/GlobalValidationSchema.tsx

```diff
@@ -41,18 +44,13 @@ Yup.setLocale({
 });
 
 export class Validation {
-  static readonly USER_PASSWORD = Yup.string().min(8).max(50).required();
-
-  static readonly USER_PASSWORD_WITH_REPEAT_NAKED = {
-    password: Validation.USER_PASSWORD,
-    passwordRepeat: Yup.string()
-      .oneOf([Yup.ref('password'), null], 'Passwords must match')
-      .required(),
-  };
-
-  static readonly USER_PASSWORD_WITH_REPEAT = Yup.object().shape(
-    Validation.USER_PASSWORD_WITH_REPEAT_NAKED
-  );
+  static readonly USER_PASSWORD = (t: TFunType) =>
+    Yup.string()
+      .min(8)
+      .max(200)
+      .required()
+      .minSymbols(1, t('password_validation_min_symbols', { count: 1 }))
+      .minNumbers(1, t('password_validation_min_numbers', { count: 1 }));
```

Comment:
It might be desirable to introduce a check for this in the backend as well; not necessary but doesn't do harm.

Reply:
Mybe later. 🤘